README.md

```diff
@@ -173,6 +173,94 @@ crud.insert_object('customers', {
 ...
 ```
 
+### Insert many
+
+```lua
+-- Batch insert tuples
+local result, err = crud.insert_many(space_name, tuples, opts)
+-- Batch insert objects
+local result, err = crud.insert_object_many(space_name, objects, opts)
+```
+
+where:
+
+* `space_name` (`string`) - name of the space to insert an object
+* `tuples` / `objects` (`table`) - array of tuples/objects to insert
+* `opts`:
+  * `timeout` (`?number`) - `vshard.call` timeout (in seconds)
+  * `fields` (`?table`) - field names for getting only a subset of fields
+  * `stop_on_error` (`?boolean`) - stop on a first error and report errors
+    regarding the failed operation and all not performed ones., default is
+    `false`
+  * `rollback_on_error` (`?boolean`) - any failed operation will lead to
+    rollback on a storage, where the operation is failed, default is
+    `false`
+
+Returns metadata and array contains inserted rows, array of errors
+(one error corresponds to one replicaset for which the error occurred).
+Error object can contain `tuple` field. This field contains the tuple
```

Comment:
Changed:

```diff
Returns metadata and array contains inserted rows, array of errors
-(one error corresponds to one replicaset for which the error occurred).
+(one error corresponds to one replicaset for which the error occurred
+if `stop_on_error` is `true`).
```

Reply:
And what's if `stop_on_error == true`?

Reply:
I updated this section:

```
Returns metadata and array contains inserted rows, array of errors.
Error object can contain field `operation_data` field.

This field can contain:
* tuple for which the error occurred;
* object with an incorrect format;
* tuples that were rollback;
* tuples that were not performed
  because operation was stopped by error.
```

Reply:
Seems resolved.